js/setup.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+var NAME = ['Иван', 'Хуан Себастьян', 'Мария', 'Кристоф', 'Виктор', 'Юлия', 'Люпита', 'Вашингтон'];
+var SURNAME = ['да Марья', 'Верон', 'Мирабелла', 'Вальц', 'Онопко', 'Топольницкая', 'Нионго', 'Ирвинг'];
+var COAT_COLOR = ['rgb(101, 137, 164)', 'rgb(241, 43, 107)', 'rgb(146, 100, 161)', 'rgb(56, 159, 117)', 'rgb(215, 210, 55)', 'rgb(0, 0, 0)'];
+var EYES_COLOR = ['black', 'red', 'blue', 'yellow', 'green'];
+var NUMBER_OF_WIZARDS = 4;
+
+var userDialog = document.querySelector('.setup');
+
+var similarListElement = userDialog.querySelector('.setup-similar-list');
+
+var similarWizardTemplate = document.querySelector('#similar-wizard-template')
+  .content
+  .querySelector('.setup-similar-item');
+
+var getRandomNumber = function (min, max) {
+  var rand = min + Math.random() * (max + 1 - min);
+  return Math.floor(rand);
+};
+
+var getRandomWizard = function (name, surname, coatColor, eyesColor) {
+  var wizardData = {
+    fullName: name[getRandomNumber(0, name.length - 1)] + ' ' + surname[getRandomNumber(0, surname.length - 1)],
+    coat: coatColor[getRandomNumber(0, coatColor.length - 1)],
+    eyes: eyesColor[getRandomNumber(0, eyesColor.length - 1)]
+  };
+
+  return wizardData;
+};
+
+var renderWizard = function (wizard) {
+  var wizardElement = similarWizardTemplate.cloneNode(true);
+
+  wizardElement.querySelector('.setup-similar-label').textContent = wizard.fullName;
+  wizardElement.querySelector('.wizard-coat').style.fill = wizard.coat;
+  wizardElement.querySelector('.wizard-eyes').style.fill = wizard.eyes;
+
+  return wizardElement;
+};
+
+var getRenderedRandomWizards = function () {
```

Comment:
Лучше
```suggestion
var getRandomWizardsNodeArr = function () {
```
Хорошая практика использовать слово `node` в названии, когда идет работа с DOM узлами. Также в названии хорошо упоминать, что функция возвращает или что переменная в себе хранит пр.: `arr`, `int`, `str`, `obj`. 

Так в будущем будет удобнее с ними работать, понимая из названия что внутри сущности